Much thanks to Richard Neumann <r(dot)neumann(at)homeinfo(dot)de> for this patch. -bts
===================================================================
--- a/networking/udhcp/dhcpc.c	2017-08-04 11:24:43.813167030 +0200
+++ b/networking/udhcp/dhcpc.c	2017-08-04 11:27:18.137996999 +0200
@@ -26,13 +26,8 @@
 #include "dhcpc.h"
 
 #include <asm/types.h>
-#if (defined(__GLIBC__) && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1) || defined(_NEWLIB_VERSION)
-# include <netpacket/packet.h>
-# include <net/ethernet.h>
-#else
-# include <linux/if_packet.h>
-# include <linux/if_ether.h>
-#endif
+#include <linux/if_packet.h>
+#include <linux/if_ether.h>
 #include <linux/filter.h>
 
 /* "struct client_config_t client_config" is in bb_common_bufsiz1 */
